app/src/main/res/values/strings.xml

```diff
@@ -186,6 +187,7 @@
     <string name="url_federation_support" translatable="false">https://support.wire.com/hc/categories/4719917054365-Federation</string>
     <string name="url_create_account_learn_more" translatable="false">https://support.wire.com/hc/articles/115004082129</string>
     <string name="url_android_release_notes" translatable="false">https://medium.com/wire-news/android-updates/home</string>
+    <string name="url_maps_location_coordinates" translatable="false">http://maps.google.com/maps?z=%1d&amp;q=loc:%2f+%2f</string>
```

Comment:
Sounds like a good idea, I'm going to timebox it, since this is the first iteration only. And in any case the url will open in the browser in case there is no google services. 

There is still [more to come](https://wearezeta.atlassian.net/browse/WPB-5169) after in the sharing location feature.

Reply:
@saleniuk Done :D 

https://github.com/wireapp/wire-android/assets/5806454/38f59262-b3d8-41e2-8371-a63a0add8314